### PR TITLE
Faq design fix

### PR DESCRIPTION
**Pull request by kozr**
Removed transition until we learn to fix it. Also, likely need to ask design team about changes for the borders of faq.

### CODE REVIEW

**Comment by Ciuffi** [comment]
We should keep the feature flag for FAQ. I'll set it to true on firebase so that the FAQ will show up, but we want to let them enter questions before we active that FAQ

**Review by Ciuffi — APPROVED** [review]
Its not perfect, still kinda janky, but hopefully its good enough.. Not really sure what else to do about it

**Comment by luceliu** [comment]
should this be closed? @Ciuffi

**Comment by Ciuffi** [comment]
@luceliu No, we have new orders from design. We should remove the squiggles and just have normal css squares instead of the svgs.

**Comment by acchiang** [comment]
are these changes on the firebase staging site? i can do cross-browser testing and review/approval for you

**Review by Ciuffi — CHANGES_REQUESTED** [review]
check the issue for new changes on the FAQ section

File: components/Faq.vue [hunk 1]
```diff
@@ -141,7 +141,7 @@ export default {
   margin: 10px auto;
   text-align: left;
   outline: none;
-  transition: 0.4s;
+  transition: 0.2s;
```
Comment (ianmah) [inline components/Faq.vue]: I don't really like the transition. Looks buggy to me. I suggest either fixing it so it's not buggy or removing it altogether

The box when it appears is cut off from the original button, which is why it looks buggy. Chrome desktop

**Review by ianmah — CHANGES_REQUESTED** [review]

File: components/Faq.vue (hunk 1 above)
Comment (Ciuffi) [reply components/Faq.vue]: Design asked us to change a lot about the FAQ, which hasn't been implemented yet. This should look better once the changes are made.

Comment (ianmah) [reply components/Faq.vue]: if you run this branch it has the new changes im pretty sure

Comment (ianmah) [reply components/Faq.vue]: image

**Comment by kozr** [comment]
image

File: components/Faq.vue (hunk 1 above)
Comment (Ciuffi) [reply components/Faq.vue]: oop you right, my bad

**Review by ianmah — APPROVED** [review]
lgtm

**Review by Ciuffi — APPROVED** [review]
Thanks nick!

**Pull request merged**